pkgs/development/python-modules/pygti/default.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+, setuptools-scm
+, aiohttp
+, pytz
+, voluptuous
+}:
+
+buildPythonPackage rec {
+  pname = "pygti";
+  version = "0.9.2";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "vigonotion";
+    repo = "pygti";
+    rev = "v${version}";
+    sha256 = "0zqa2krsniaqisjr0xqw009wdyy3y48zar9lrwysjsqci7k07d9x";
+  };
+
+  nativeBuildInputs = [
+    setuptools-scm
+  ];
+
+  SETUPTOOLS_SCM_PRETEND_VERSION = version;
+
+  propagatedBuildInputs = [
+    aiohttp
+    pytz
+    voluptuous
+  ];
+
+  # no tests implemented
+  doCheck = false;
+
+  pythonImportsCheck = [
+    "pygti.auth"
+    "pygti.exceptions"
+    "pygti.gti"
+  ];
+
+  meta = with lib; {
+    description = "Access public transport information in Hamburg, Germany";
+    homepage = "https://github.com/vigonotion/pygti";
+    license = licenses.mit;
```

Comment:
https://github.com/vigonotion/pygti/blob/master/setup.py#L110
MIT

https://github.com/vigonotion/pygti/blob/master/LICENSE.md
ASL20

Needs clarification.

Reply:
It's definitely the latter: https://github.com/vigonotion/pygti/blob/master/CONTRIBUTING.md#any-contributions-you-make-will-be-under-the-apache-license-20

I made an upstream PR: https://github.com/vigonotion/pygti/pull/70

Reply:
Okay, just opened an issue as well, so maybe you can add a close marker.

https://github.com/vigonotion/pygti/issues/71